libexec/rtld-elf/rtld_paths.h

```diff
@@ -74,6 +77,18 @@
 #define	LD_			"LD_64_"
 #endif
 
+#ifndef _PATH_ELF_C18N_HINTS
+#define	_PATH_ELF_C18N_HINTS	"/var/run/ld-elf-c18n.so.hints"
```

Comment:
@brooksdavis The hints file referred to by this path is missing, which seems to result in libraries located in `/usr/local/lib` being unresolvable.

Should this simply be a duplicate of `/var/run/ld-elf.so.hints`, at least for now? The file seems to be a special binary file, and I'm not sure how to generate it.

Reply:
Hm, probably, we don't really want an `ldconfig -c18n` (especially since pkg64c isn't going to trigger that) nor do we need it